multilep/test/crab.py

```diff
@@ -6,13 +6,12 @@
 dataset         = os.environ['CRAB_DATASET']
 outputFile      = os.environ['CRAB_OUTPUTFILE']
 lumiMask        = os.environ['CRAB_LUMIMASK']
+extraContent    = os.environ['CRAB_EXTRACONTENT']
 
 requestName = dataset.split('/')[2] + '_' + productionLabel
-requestName = requestName.replace('RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6','Moriond2017')
-requestName = requestName.replace('RunIISummer16MiniAODv2-PUMoriond17_HCALDebug_80X_mcRun2_asymptotic_2016_TrancheIV_v6','Moriond2017')
-requestName = requestName.replace('RunIISummer16MiniAODv2-80X_mcRun2_asymptotic_2016_TrancheIV_v6', '80X')
-requestName = requestName.replace('RunIISpring16MiniAODv2-PUSpring16Fast_80X_mcRun2_asymptotic_2016', 'Spring16FS')
-requestName = requestName.replace('RunIISummer16MiniAODv2-PUSummer16Fast_80X_mcRun2_asymptotic_2016_TrancheIV_v6', 'Moriond2017FS')
+requestName = requestName.replace('RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_v3', 'MiniAOD2016v3')
+requestName = requestName.replace('RunIIFall17MiniAODv2-PU2017_12Apr2018_94X_mc2017_realistic_v14', 'MiniAOD2017')
+requestName = requestName.replace('RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15', 'MiniAOD2018')
```

Comment:
Adding a temporary replacement for RunIISummer16MiniAODv3-PUMoriond17_94X_mcRun2_asymptotic_**2016**_v3 samples might be a good idea 
(and other cases in which a sample is only available in a different version)

Reply:
- RunIIFall17MiniAODv2-PU2017_12Apr2018_new_pmx_94X_mc2017_realistic_v14 --> I guess this is ok, though it would be good if we could find out what "new_pmx" means
- PUMoriond17_94X_mcRun2_asymptotic_2016_v3 --> suggests these ones are with buggy pile-up (as "PU2017" is used to denote where the PU is fixed), but apparently there was also some bug in how samples were named so we can actually not trust any naming for 2017, it would be good to find documentation, but no idea where to look for it
- RunIIAutumn18MiniAOD-PUAvg50ForMUOVal_102X_upgrade2018_realistic_v15_ext4-v1 --> looks like some flat PU distribution, I realize we should not use the sample and delete it from the list
 RunIIAutumn18MiniAOD-NZSFlatPU28to62_102X_upgrade2018_realistic_v15_ext5-v1 --> idem

Reply:
Found statements like "new_pmx samples with corrected PU shape",  in addition to lots of commits about switching to new_pmx samples from the normal ones (if available).
Best guess: any "PU2017" sample for which a "new_pmx" sample appears is bad, 
so switch to the new_pmx version whenever one is available